src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -1529,7 +1529,7 @@ Connection connectInternal(Properties propsIn,
             }
             authenticationString = SqlAuthentication.valueOfString(sPropValue).toString();
 
-            if ((true == integratedSecurity)
+            if ((integratedSecurity)
```

Comment:
Unnecessary brackets around `integratedSecurity` and `!authenticationString.equalsIgnoreCase(SqlAuthentication.NotSpecified.toString()`